module/extension/Director/src/Director.cpp

```diff
@@ -31,6 +31,8 @@ namespace module::extension {
     using ::extension::behaviour::commands::ProvidesReaction;
     using ::extension::behaviour::commands::WhenExpression;
     using ::NUClear::threading::Reaction;
+    using provider::Provider;
```

Comment:
Formatting?

Reply:
I don't understand what you're mad about